docs/notebooks/bsync_examples/Small-Office-Level-2.md

```diff
@@ -0,0 +1,2179 @@
+
+# 6.1.1.1.m 
+b1 += bsync.PremisesNotes("Here we record general problems / issues identified in a walkthrough survey.")
+
+# 6.1.1.1.d - address
+b1 += bsync.Address(
+    bsync.StreetAddressDetail(
+        bsync.Simplified(
+            bsync.StreetAddress("Some address")
+        )
+    ),
+    bsync.Address.State("CO"),
+    bsync.City("Denver"),
+    bsync.PostalCode("80401")
+)
+
+# 6.2.1.1.b - gross and conditioned floor area
+b1 += bsync.FloorAreas(
+    bsync.FloorArea(
+        bsync.FloorAreaType("Gross"),
+        bsync.FloorAreaValue(5500.0),
+        bsync.ExcludedSectionIDs(
+            bsync.ExcludedSectionID(IDref="Section-excluded")
+        )
+    ),
+    bsync.FloorArea(
+        bsync.FloorAreaType("Conditioned"),
+        bsync.FloorAreaValue(5500.0),
+        bsync.ExcludedSectionIDs(
+            bsync.ExcludedSectionID(IDref="Section-excluded")
+        )
+    )
+)
+
+# 6.2.1.2.b - gross wall area
+b1 += bsync.TotalExteriorAboveGradeWallArea(6500.)
+b1 += bsync.TotalExteriorBelowGradeWallArea(0.)
+
+# 6.2.1.2.c - fenestration-wall ratio
+b1 += bsync.OverallWindowToWallRatio(0.2133)
+b1 += bsync.OverallDoorToWallRatio(0.05)
+
+# 6.1.1.1.f - classification of the uses
+b1 += bsync.BuildingClassification("Commercial")
+b1 += bsync.OccupancyClassification("Office")
+b1 += bsync.BuildingAutomationSystem(True)
+b1 += bsync.HistoricalLandmark(False)
+
+# 6.1.1.1.h - for completeness, we spell this out explicitly
+b1 += bsync.FloorsAboveGrade(1)
+b1 += bsync.FloorsBelowGrade(0)
+b1 += bsync.ConditionedFloorsAboveGrade(1)
+b1 += bsync.ConditionedFloorsBelowGrade(0)
+
+# 6.1.1.1.i - again, to be explicit, we define the latest remodel / retrofit year
+# to be the same as the original build year to indicate nothing has happened since original construction
+b1 += bsync.YearOfConstruction(2000)
+b1 += bsync.YearOccupied(2000)
+b1 += bsync.RetrocommissioningDate(date(2020,3,1))
+b1 += bsync.YearOfLastMajorRemodel(2000)
+b1 += bsync.YearOfLastEnergyAudit(2010)
+
+#pretty_print(root)
+```
+
+Continuing with the information required by 6.1.1.1, we define additional content that doesn't sit as direct child elements of the Building. This includes:
+- Contacts. Key contacts (1 auditor and 1 owner) __must be linked__ back to the building, even though they are not in the Building subtree:
+    - Contacts live: /BuildingSync/Facilities/Facility/Contacts/Contact
+    - An auditor is linked to a building at a Report level (i.e. who performed the audit)
+    - Reports live: /BuildingSync/Facilities/Facility/Reports/Report
+    - An audit report, with the correct links to a Building, Measures, etc. should be sufficient to meet the Standard 211 requirements spelled out in Section 6. More on this later.
+- Space function breakdown by space type
+- Occupied hours and number of occupants
+
+#### Contacts
+
+
+```python
+# 6.1.1.1.b - Owner and auditor contact info
+c1 = bsync.Contact(
+    bsync.ContactName('The dude'),
```

Comment:
We are mixing double quotes and single quotes, pick one. I prefer single quotes for most python strings.

Reply:
Based on our conversation, we will update the whole file to have double quotes as xml prefers that, but let's keep this PR in single quotes for now.